include/verilated_vpi.cpp

```diff
@@ -2447,6 +2550,17 @@ vpiHandle vpi_put_value(vpiHandle object, p_vpi_value valuep, p_vpi_time /*time_
         return nullptr;
     }
     if (const VerilatedVpioVar* const vop = VerilatedVpioVar::castp(object)) {
+        if (flags & vpiInertialDelay) {
```

Comment:
My reading of IEEE is this isn't a bit mask but absolute number.  (If not sure look at Icarus?)

```suggestion
        if (flags == vpiInertialDelay) {
```

Reply:
> My reading of IEEE is this isn't a bit mask but absolute number. (If not sure look at Icarus?)

Super weird.  `flags` is both an enum value and a bit vector combined.  I think the only bit vector field that applies to `vpi_put_value()` is `vpiReturnEvent`.  I don't see a formal definition of what to mask off when comparing the enum portion of `flags`, but from looking at `vpi_user.h`, I think it is probably `0xfff`.

I'm not sure I agree with the LRM's use of the term `flag` here, but oh well.